examples/lfric/eg17/full_example_extract/main.x90

```diff
@@ -40,7 +40,7 @@ program main
     use function_space_mod,     only: function_space_type
     use fs_continuity_mod,      only: W0, W1, W2, W2V, W2H
     use field_mod,              only: field_type
-    use testkern_w0_mod,        only: testkern_w0_type
+    use testkern_w0_kernel_mod, only: testkern_w0_kernel_type
```

Comment:
I think the copyright years should be 2020-2021 as the file is now modified.

Reply:
Done.